libindy/src/api/wallet.rs

```diff
@@ -308,6 +308,132 @@ pub extern fn indy_list_wallets(command_handle: i32,
     res
 }
 
+/// Exports opened wallet
+///
+/// #Params:
+/// handle: wallet handle returned by indy_open_wallet
+/// export_config_json: JSON containing settings for input operation.
+///   {
+///     "path": path of the file that contains exported wallet content
+///     "key": passphrase used to export key
+///   }
+///
+/// #Returns
+/// Error code
+///
+/// #Errors
+/// Common*
+/// Wallet*
+#[no_mangle]
+pub extern fn indy_export_wallet(command_handle: i32,
+                                 handle: i32,
```

Comment:
> handle

Rename it to wallet_handle

Reply:
i used existing conventions from close_wallet call